tracee-ebpf/tracee/tracee.go

```diff
@@ -72,6 +73,39 @@ type netProbe struct {
 	ingressHook *bpf.TcHook
 	egressHook  *bpf.TcHook
 }
+func getAddrFromKallsyms(func_name string) uint64 {
```

Comment:
we can return here error

Reply:
i added a new definition with error and uint64 as return vals